src/server/migrations/20201129124100_create_groups_schema.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function(knex) {
+  return knex.schema.createTable('groups', (table) => {
+    table.increments();
+    table.string('title').notNullable();
```

Comment:
This column should be called "name" instead ?  Good job with the rest.

![image](https://user-images.githubusercontent.com/61273043/100674648-c719c380-3365-11eb-8d19-41cd199abbd1.png).